app/lib/actions/pulp3/orchestration/repository/generate_metadata.rb

```diff
@@ -7,9 +7,9 @@ def plan(repository, smart_proxy, options = {})
             options[:contents_changed] = (options && options.key?(:contents_changed)) ? options[:contents_changed] : true
             sequence do
               plan_action(Actions::Pulp3::Repository::CreateVersion, repository, smart_proxy) if options[:repository_creation]
-              if options[:source_repository]
+              if options[:source_repository] && repository.content_type != "docker"
                 repository.update_attributes!(publication_href: options[:source_repository].publication_href)
-              else
+              elsif repository.content_type != "docker"
```

Comment:
we're trying to make content types be as 'pluggable' as possible, so we might want to add this logic to the repository type registration

Reply:
I'm going to wait and see what we decide for the Ansible PR since it has similar changes being made.